src/pybind/mgr/dashboard/controllers/_crud.py

```diff
@@ -64,7 +64,8 @@ class TableAction(NamedTuple):
     name: str
     permission: str
     icon: str
-    routerLink: str  # redirect to...
+    routerLink: str = ''  # redirect to...
+    click: str = ''
```

Comment:
Why an extra `click` variable? Correct me if I am wrong but if this is for assuming the action that is going to perform (create, edit, delete etc), then the `name` itself should be enough right?

Reply:
The idea is to have a variable, `click`, that you can set the name to match your frontend function, and then to map this click variable into the `CdTableAction` `click` variable that is generated from the `meta`. 

I thought this was a good approach to have functions that don't use a form, and they just trigger a function, let me know what you think Nizam